2015/P_U2/main.cpp

```diff
@@ -0,0 +1,95 @@
+#include <iostream>
+#include <fstream>
+#include <vector>
+using namespace std;
+
+struct Mokinys{
+  string vardas;
+  string atsakymai;
+  int taskuKiekis = 0;
+  bool mokytojas = false;
```

Comment:
Do we really need that? Maybe we can define this as Participant, and have a list of students and the object of teacher?